lib/services/http-service.js

```diff
@@ -218,6 +219,11 @@ function httpServiceFactory(
             )
         );
 
+        //re-route common and current
+        var versionPath = configuration.get('versionBase') || '1.1';
```

Comment:
@keedya I think we should add an assert or some other type of check to verify that `versionBase` is set to a valid value.